packages/web-components/src/components/cbp-tag/cbp-tag.scss

```diff
@@ -26,33 +26,33 @@ cbp-tag {
   cursor: default;
   padding: 0 var(--cbp-space-3x);
   text-align: center;
-  font-weight: var(--cbp-font-weight-medium);
+  font-weight: var(--cbp-font-size-subhead);
```

Comment:
this is incorrect. This variable is a font size, which needs to be specified properly.

Reply:
The old font-weight is still needed though.